README.md

```diff
@@ -114,6 +114,42 @@ If you want to customize the icons or the theme, refer to the [customization sec
 <script src="https://unpkg.com/buefy/dist/buefy.min.js"></script>
 ```
 
+
+### Installing Developer Snapshots from GitHub npm Registry to Access the Latest Features
```

Comment:
Do we call packages in GitHub npm Registry "developer releases"? If so, please replace "developer snapshot(s)" with "developer release(s)".

Reply:
> Do we call packages in GitHub npm Registry "developer releases"? If so, please replace "developer snapshot(s)" with "developer release(s)".

Regarding the npm packages for `Buefy-next`, I want to clarify that they are technically packages of each release made to the dev branch on our GitHub npm registry. These are not developer snapshots, but rather developer releases. 

Thank you for bringing this to my attention! I will make the necessary changes when I have some free time.